gneiss/cluster/_cluster.py

```diff
@@ -121,3 +123,31 @@ def gradient_clustering(table: pd.DataFrame,
                  'This method is primarily used to sort the table to reveal '
                  'the underlying block-like structures.')
 )
+
+
+def assign_ids(tree: skbio.TreeNode) -> skbio.TreeNode:
+    def replace_dashes(x):
+        return str(x).replace('-', '')
+    t = tree.copy()
+    t.bifurcate()
+    ids = ['%sL_%s' % (i, replace_dashes(uuid.uuid4()))
```

Comment:
if you do this, you can't test to see if the id is a uuid. are you sure you want to replace the dashes?